hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/impl/BackupSystemTable.java

```diff
@@ -1639,7 +1642,7 @@ public String[] getListOfBackupIdsFromDeleteOperation() throws IOException {
       if (val.length == 0) {
         return null;
       }
-      return new String(val).split(",");
+      return new String(val, StandardCharsets.UTF_8).split(",");
```

Comment:
I think this will introduce another error prone warning? Use guava's Splitter.on

Reply:
It does not.